### PR TITLE
rebuild project structure 

### PR DESCRIPTION
**Pull request by baoandlk93**
#add GuestLayout
#add "react-icons": "^4.12.0"
#move directory page to directory layouts
#define AdminHeader
#define Login component
#define Register component
#define Sidebar component
#define Navbar component

### CODE REVIEW

**Review by nguyenthanhdat1997 — APPROVED** [review]
good job!

**Review by thinhphan1122 — APPROVED** [review]
Approved.

**Review by phaiphan — APPROVED** [review]

File: src/redux/api/UserApi.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import axios from "axios";
-import {SHOP_API} from "../../ultility/AppConstant.js"
+import {WEB_API} from "../../ultility/AppConstant.js"
```
Comment (daonguyendev) [inline src/redux/api/UserApi.js:2]: Should rename WEB_API to BEST_TICKET_API

**Review by daonguyendev — CHANGES_REQUESTED** [review]
Please update some codes according to the comments below

File: src/component/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    const handleToogle = (e)=>{
+        if (e.target.checked){
+            setTheme("dark")
+        }else {
+            setTheme("light")
+        }
+    }
+
+    useEffect(() => {
+        localStorage.setItem("theme", theme)
```
[the remark this answers is not in the document]
Comment (baoandlk93) [reply src/component/Navbar.jsx:18]: Thank you for your comment, I will fix it in the next commit !

**Review by NhanTran262 — APPROVED** [review]
Well done

**Review by baoandlk93 — COMMENTED** [review]
Everything ok !

**Review by phaiphan — APPROVED** [review]

**Review by nguyenthanhdat1997 — COMMENTED** [review]
good job

**Review by NhanTran262 — APPROVED** [review]
Well done

**Review by phaiphan — APPROVED** [review]
good job

**Review by thinhphan1122 — APPROVED** [review]
Reviewed

**Review by thinhphan1122 — COMMENTED** [review]
Re-reviewed

**Review by baoandlk93 — COMMENTED** [review]
Reviews done

**Review by thinhphan1122 — APPROVED** [review]
Re-approve

**Review by nguyenthanhdat1997 — COMMENTED** [review]
nice

**Review by NhanTran262 — APPROVED** [review]
GLTM

**Review by nguyenthanhdat1997 — APPROVED** [review]
LGTM!

**Review by nguyenthanhdat1997 — COMMENTED** [review]
LGTM!

**Review by thinhphan1122 — APPROVED** [review]
Approved!

**Review by NhanTran262 — APPROVED** [review]
GLTM

**Review by phaiphan — APPROVED** [review]
good

**Pull request merged**